packages/spruce-skill-server/package.json

```diff
@@ -23,9 +23,8 @@
 		"build": "npm run clean && tsc | true",
 		"watch": "tsc -w -p ./tsconfig.json",
 		"clean": "rm -rf build/",
-		"#lint": "eslint '**/*.ts' '**/*.js'",
 		"lint": "eslint '**/*.ts' && tsc -p . --noEmit",
-		"test": "cd ../spruce-skill/ && rm tmp/testing.db | true && mkdir -p tmp/ | true && TS_NODE_PROJECT=../spruce-skill-server/tsconfig.json PORT=3010 BLUEBIRD_W_FORGOTTEN_RETURN=0 ALLOW_NON_SSL=true ORM_LOGGING=false TESTING=true API_ONLY=true DB_MIGRATIONS=false mocha -r ts-node/register/transpile-only --exit '../spruce-skill-server/tests/**/*Tests.ts' --inspect=4080",
+		"test": "cd ../spruce-skill/ && rm tmp/testing.db | true && mkdir -p tmp/ | true && TS_NODE_PROJECT=../spruce-skill-server/tsconfig.json DEBUG=* PORT=3010 BLUEBIRD_W_FORGOTTEN_RETURN=0 ALLOW_NON_SSL=true ORM_LOGGING=false TESTING=true TESTING_SKILLS_KIT=true API_ONLY=true DB_MIGRATIONS=false mocha -r ts-node/register/transpile-only --exit '../spruce-skill-server/tests/**/*Tests.ts' --inspect=4080",
```

Comment:
`TESTING_SKILLS_KIT=true` should be added to the workspace `launch.json` file as an env var so we can test individual example files